### ISSUE TITLE
Metallic and Roughness workflow

### ISSUE THREAD

**Issue by PlasmaDev5** [TheCherno/Sparky#110]
Many other renderers that use PBR look to work on a Metallic and Roughness workflow, i feel adding this to Sparky could be useful for people coming from these engines along with many assets being made for Metallic and Roughness workflows.

P.S im happy to look into implementing.

**Comment by TheCherno** [comment]
Yes, supporting metalness (instead of specular) would be nice from a compatibility point of view. It will need a slightly different shader though, so feel free to look into that if you like.

**Comment by PlasmaDev5** [comment]
i ran into issues converting the metallic and albedo into the specular within the shader.

**Comment by jannisj1** [comment]
@dragonCASTjosh  Did you use [this](http://www.marmoset.co/toolbag/learn/pbr-conversion)?
